src/commands/CommandClear.java

```diff
@@ -175,7 +226,14 @@ public String getCommandDescription(){
 		{
 			new ParametersHelp(
 					"Waits that the message has been successfully deleted before deleting the others. Useful if you are not sure if you should delete all the messages as you can stop the command.",
-					false, "s", "slow")
+					false, "i", "interactive"),
+			new ParametersHelp(
+				"Allows you to delete your own messages.",
+				false, "s", "self"),
+			new ParametersHelp(
+				"Allows you to delete all of the bots messages.",
+				false, "b", "bot"),
+			
```

Comment:
Just realized that you are also missing the `-u` / `--user` parameter help here.

Reply:
Oups forgot about that.